src/runtime/virtcontainers/hypervisor.go

```diff
@@ -65,7 +65,7 @@ const (
 
 	procCPUInfo = "/proc/cpuinfo"
 
-	defaultVCPUs = float32(1)
+	defaultVCPUs = float32(0)
```

Comment:
this change was not needed but the change in `src/runtime/pkg/katautils/config-settings.go.in` was required. I decided to leave this change though